Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/qrcode/RapidAntigenQrCodeExtractor.kt

```diff
@@ -49,7 +49,12 @@ class RapidAntigenQrCodeExtractor @Inject constructor() : QrCodeExtractor<Corona
         } else {
             BaseEncoding.base64Url().decode(this)
         }
-        return Gson().fromJson(decoded.commonToUtf8String())
+        try {
+            return Gson().fromJson(decoded.commonToUtf8String())
+        } catch (e: Exception) {
```

Comment:
`JsonSyntaxException` instead of `Exception` ?

Reply:
I prefer to catch all exception so the app does not crash if something goes wrong here

Reply:
It's about any nonsense payload that may occur